modules/021-cni-cilium/images/cilium/Dockerfile

```diff
@@ -0,0 +1,119 @@
+# syntax=docker/dockerfile:1.2
+
+# Copyright 2020-2021 Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+ARG GOLANG_IMAGE=docker.io/library/golang:1.18.0@sha256:b5a9ae5907066849cbb2b47af93713b1f030e3eb6b36d84f2ae5f354b25c80d4
+ARG UBUNTU_IMAGE=docker.io/library/ubuntu:20.04@sha256:626ffe58f6e7566e00254b638eb7e0f3b11d4da9675088f4781a50ae288f3322
```

Comment:
These are taken directly from the version 1.11.4. I'd like to avoid any problems with Cilium build dependencies by following their build patterns to the letter.